src/applications/SurvivalPredictor.cpp

```diff
@@ -707,30 +773,29 @@ VectorDouble SurvivalPredictor::SurvivalPredictionOnExistingModel(const std::str
 
 }
 
-VariableSizeMatrixType SurvivalPredictor::SelectSixMonthsModelFeatures(const VariableSizeMatrixType &SixModelFeatures)
+VariableSizeMatrixType SurvivalPredictor::SelectModelFeatures(const VariableSizeMatrixType &SixModelFeatures,const VariableLengthVectorType selectedfeatures)
 {
-   int selectedFeatures[20] = { 1,    5,    9,    10,    20,    23,    24,    37,    38,    43,    44,    48,    49,    50,    51,    56,    57,    61,    62,    63};
-
+   //int selectedFeatures[20] = { 1,    5,    9,    10,    20,    23,    24,    37,    38,    43,    44,    48,    49,    50,    51,    56,    57,    61,    62,    63};
 //   int selectedFeatures[20] = { 1, 5, 9, 10, 13, 26, 33, 35, 38, 41, 43, 44, 48, 50, 51, 56, 57, 63, 64, 69 };
+   //for (unsigned int i = 0; i <20; i++)
+   // selectedFeatures[i] = selectedFeatures[i] - 1;
```

Comment:
If the commented code is needed in future, please leave some comments about why it was commented. If not needed, please delete.

Reply:
i have removed the commented code. it is not needed in future.